### PR TITLE
Improvements to the visuals of the deck builder/tag system

### PR DESCRIPTION
**Pull request by alicetouhou**
- Colors icons to match their item card
- Changes how tags display to be more visually appealing
- Redoes styling of mobile deck builder to be more appealing
image
image

### CODE REVIEW

**Comment by Lunarmagpie** [comment]
Visuals look good, will check the code in a bit

File: client/src/app/deck/deck.module.scss [hunk 1]
```diff
@@ -167,6 +167,46 @@
...
 	}
+
+	&.farm {
+		background-color: rgb(124, 204, 12);
+	}
+
+	&.redstone {
+		background-color: rgb(185, 33, 42);
+	}
+
```
Comment (Lunarmagpie) [inline client/src/app/deck/deck.module.scss:178]: don't copy all of this CSS twice. Its definitely going to be forgot about and become desynced.

**Review by Lunarmagpie — CHANGES_REQUESTED** [review]

File: client/src/app/deck/deck.module.scss (hunk 1 above)
Comment (alicetouhou) [reply client/src/app/deck/deck.module.scss]: Oops, was going to do this, fixed

**Review by Lunarmagpie — APPROVED** [review]
LGTM

**Pull request merged**